src/pages/Favourites.css

```diff
@@ -1,5 +1,3 @@
 .container {
-  display: flex;
-  height: 30px;
-  font-size: 2rem;
+  color: #ffffff;
```

Comment:
Could this color saved in a css variable?

Reply:
Most definitely. I missed that one. Thanks for catching it!